README.md

```diff
@@ -1,58 +1,48 @@
-# Open棟梁(OpenTouryo)
-”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
-(”OpenTouryo” , is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.)
+# OpenTouryo
+"OpenTouryo", is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.
 
-## 開発／動作環境(develop/run-time environment)
-このリポジトリのプログラムは、サポートされる最も古い開発／実行環境を使用して開発しています。 (I am developing the program of this repository, using the oldest supported develop/run-time environment.)
-- 開発環境 (develop environment)
-    - Visual Studio 2010
-- 実行環境 (run-time environment)
-    - .NET Framework 3.5 SP1
+## Develop/Run-time environment
+The program in this repository is developed, using the oldest supported develop/run-time environment.
```

Comment:
@daisukenishino 

Please revise as follows:

The programs in this repository are developed in the following environments: